packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockUsers.st

```diff
@@ -1,9 +1,4 @@
 user
 mockUsers
 
-	| promise users |
-	promise := Promise new.
-	promise resolveWith: (TCCUser newFrom: self mockUserEvent).
-	users := TCCUsers new.
-	users at: self mockUser id put: promise.
-	^ users
\ No newline at end of file
+	^ {self mockUser1 . self mockUser2}
```

Comment:
How about `mockChatMembers`. That would possibly resolve the naming conflict with `TCCUsers` and `mockUserPromises` as mentioned in https://github.com/hpi-swa-teaching/TelegramClient/pull/394/files#r651013534.

Reply:
the conflict will be resolved when we rename TCUUsers